### PR TITLE
Remove import for AbortController

### PR DESCRIPTION
**Pull request by milind12**

### CODE REVIEW

**Comment by milind12** [comment]
Please update index.d.ts as per https://github.com/node-fetch/node-fetch/blob/master/index.d.ts

**Comment by southpolesteve** [comment]
@milind12 Can you elaborate on why this change is needed? The master branch of `node-fetch` currently points to their v3 release which is not yet latest on npm. They will ship built-in type definitions and the @types/node-fetch will no longer be needed.

At a minimum, I think we should keep this package inline with node-fetch v2

**Comment by milind12** [comment]
Can you please refer this issue https://github.com/node-fetch/node-fetch/issues/741 . 
'onabort' in AbortSignal interface is inconsistent with 'onabort' in actual DOM AbortSignal interface

File: types/node-fetch/externals.d.ts [hunk 1]
```diff
@@ -17,5 +17,5 @@ export interface AbortSignal {
 
     dispatchEvent: (event: any) => boolean;
 
-    onabort?: null | ((this: AbortSignal, event: any) => void);
+    onabort: ((this: AbortSignal, ev: Event) => any) | null;
```
Comment (southpolesteve) [inline types/node-fetch/externals.d.ts:20]: I believe this is testable. In the test file, reference `lib.dom.ts` at the top and then add a new test passing in the global `AbortSignal` to node-fetch.

Comment (milind12) [reply types/node-fetch/externals.d.ts:20]: Hi @southpolesteve , I am closing this MR as I found a way around

Comment (ttrepka) [reply types/node-fetch/externals.d.ts:20]: @milind12 What is the way around, please? I'm still facing the same issue (https://github.com/node-fetch/node-fetch/issues/741) which your PR might solve.

Comment (milind12) [reply types/node-fetch/externals.d.ts:20]: @ttrepka please check there is one http option to set the timeout, (not a recommended way) but it has always worked for me. https://www.npmjs.com/package/node-fetch#options

Comment (ttrepka) [reply types/node-fetch/externals.d.ts:20]: Thanks, I would rather use the recommended way. I guess I can do this for now but would be nicer if I didn't:
```
fetch(url, { signal: abortController.signal as any })
```

**Pull request closed without merging**